printer_mods/ByteChild/2_Channel_Relay_DIN_Rail_Bracket/Readme.md

```diff
@@ -0,0 +1,9 @@
+This is a DIN rail bracket for an 2-channel relay bought from [Amazon](https://www.amazon.de/-/en/gp/product/B00PIMRGN4/)<br>
+<br>
+May also fit for other 2-channel relays, but I have not tested.<br>
+The spacing for the bores is: 33mm x 45mm<br>
+<br>
+Please use the generic PCB DIN Clip and mount this on top.<br>
+<br>
```

Comment:
Please remove the space from the `2_Channel_Relay_DIN_Rail_Bracket/relay _bracket.step` filename